doc/ompl_interface/ompl_interface_tutorial.rst

```diff
@@ -65,6 +65,9 @@ Several planners that are part of the OMPL planning library are capable of optim
 * SPARS
 * SPARS2
 * Transition-based RRT (T-RRT)
+* AIT*
+* BIT*
+* ABIT*
```

Comment:
Also, I am not sure what is the common standard here (some planners have only the abbreviation, while others have the full name as well), but I personally think writing the full name would be better.

```suggestion
* Batch Informed Trees (BIT*)
* Advanced BIT* (ABIT*)
* Adaptively Informed Trees (AIT*)
```

Reply:
Got it.